### DISCUSSION TITLE
Cannot start the app

### DISCUSSION THREAD

**Discussion by skret028** [photoprism/photoprism#3795] [Q&A]
I'm running photoprism docker on Unraid 6.12.4.
It worked fine for a year,but recently starts to report some errors and cannot start the app.

Container Logs:

> started 230625 as root (amd64-prod)
Problems? Our Troubleshooting Checklists help you quickly diagnose and solve them:
https://docs.photoprism.app/getting-started/troubleshooting/
file umask....: "0002" (u=rwx,g=rwx,o=rx)
home directory: /photoprism
assets path...: /opt/photoprism/assets
storage path..: /photoprism/storage
config path...: default
cache path....: default
backup path...: /photoprism/storage/backups
import path...: /photoprism/import
originals path: /photoprism/originals
running as uid 0
/opt/photoprism/bin/photoprism start
/opt/photoprism/bin/photoprism: error while loading shared libraries: libstdc++.so.6: cannot open shared object file: No such file or directory

My docker test scripts,copied from official docs:

```
docker run -d \
  --name photoprism \
  --security-opt seccomp=unconfined \
  --security-opt apparmor=unconfined \
  -p 2342:2342 \
  --net='ipvlan_net' \
  --ip='192.168.10.10' \
  -e PHOTOPRISM_UPLOAD_NSFW="true" \
  -e PHOTOPRISM_ADMIN_PASSWORD="insecure" \
  -v /mnt/user/container_config/photoprism:/photoprism/storage \
  -v /mnt/user/photoprism/originals:/photoprism/originals \
  photoprism/photoprism

```

**Comment by lastzero** [comment]
Does it work when you pull our most recent Docker image? Did you try checking solutions to that error message on Google yet? I've never seen it, so it might not be related to our software specifically.
